admin/src/main/resources/io/buoyant/admin/js/router_server.js

```diff
@@ -18,7 +18,7 @@ var RouterServer = (function() {
         metricSuffix: "success",
         query: Query.serverQuery().withRouter(routerName).withServer(serverName).withMetric("success").build(),
         getRate: function(data) {
-          var successRate = new SuccessRate(data.requests, data.success, data.failures);
+          var successRate = new SuccessRate(data.success, data.failures);
```

Comment:
data.failures isn't always present in the returned metrics (if there haven't yet been any failures) so sometimes the dashboard will display NaN% where it should display N/A.  We should (I can do this later) do some checks on data.failures.